src/store/debug/debug-reducers.ts

```diff
@@ -0,0 +1,13 @@
+import { createReducer } from '@reduxjs/toolkit';
+
+import { pushAction } from './debug-actions';
+import { debugInitialState, DebugState } from './debug-state';
+
+const MAX_RECENT_EVENTS = 20;
+
+export const debugReducers = createReducer<DebugState>(debugInitialState, builder => {
+  builder.addCase(pushAction, (state, { payload }) => ({
+    ...state,
+    recentActions: [...state.recentActions.slice(-MAX_RECENT_EVENTS + 1), payload]
```

Comment:
`MAX_RECENT_EVENTS`, `recentActions` naming mismatch

Reply:
How do you want me to name them?

Reply:
it's up to you)
for ex. 
`MAX_RECENT_ACTIONS`, `recentActions`, `RecentActionPayload`, `pushRecentAction`